cql3/Cql.g

```diff
@@ -1545,6 +1554,20 @@ serviceLevelOrRoleName returns [sstring name]
 | QMARK {add_recognition_error("Bind variables cannot be used for service levels or role names");}
 ;
 
+serviceLevelNameExpr returns [uexpression expr]
+@init{sstring name;}
+: t=IDENT              { name = sstring($t.text);
+                         std::transform(name.begin(), name.end(), name.begin(), ::tolower);
```

Comment:
Why tolower? What are the rules for service level names?

Reply:
You can check `serviceLevelOrRoleName` function in the grammar.

Service level and role names are parsed as lower case on a grammar level unless quoted.